flytekit/annotated/constants.py

```diff
@@ -0,0 +1 @@
+RAW_CONTAINER_TASK_TYPE = "raw-container"
```

Comment:
don't we have this already somewhere else?  can we use that one @kumare3

Reply:
it was defined inline in container_task.py

Reply:
it's defined in `flytekit/common/constants.py:25`

Reply:
okay apparently this is just a console thing, ignore my last comment.

Reply:
can we not define a constant again :(, it is part of the task